### PR TITLE
azurerm minor version update

### LINKED ISSUE #5785: Upgrade Okta terraform provider to version 4

### PR DESCRIPTION
**Pull request by alismx**
# DEVOPS PULL REQUEST

## Related Issue

- resolve #5785 

## Changes Proposed

- update `azurerm` provider to `3.70.0`

## Additional Information

- Cors blocks were removed
  - This version requires `allowed_origins` not to be an empty list
  - `support_credentials` defaults to false

```
cors {
  allowed_origins     = []
  support_credentials = false
}
```

## Testing

- Smoke test dev4, this is deployed there.
- [Check the terraform deployment step for the dev4 deployment](https://github.com/CDCgov/prime-simplereport/actions/runs/5979854313/job/16224866661)
- Run [ad-hoc terraform plans](https://github.com/CDCgov/prime-simplereport/actions/workflows/terraformPlan.yml) with this branch against any environment to validate that nothing really changes
- [dev plan](https://github.com/CDCgov/prime-simplereport/actions/runs/5980204851/job/16225850541)
- [prod plan](https://github.com/CDCgov/prime-simplereport/actions/runs/5980237631)

### Infrastructure
- [ ] Consult the results of the `terraform-plan` job inside the "Terraform Checks" workflow run for this PR. Confirm that there are no unexpected changes!

### Security
- [ ] Changes with security implications have been approved by a security engineer (changes to  authentication, encryption, handling of PII, etc.)
- [ ] Any dependencies introduced have been vetted and discussed

### Cloud
- [ ] Oncall has been notified if this change is going in after-hours
- [ ] If there are changes that cannot be tested locally, this has been deployed to our Azure `test`, `dev`, or `pentest` environment for verification

### Documentation
- [ ] Any changes to the startup configuration have been documented in the README

### CODE REVIEW

**Review by fzhao99 — APPROVED** [review]
lgtm!

File: ops/upgradetf.sh [hunk 1]
```diff
@@ -12,5 +12,5 @@ terraform fmt -recursive
 for d in $TERRAFORM_DIRS
 do
     echo "Initializing/formatting/validating $d";
-    (cd $d && rm .terraform.lock.hcl && terraform init -upgrade -backend=false && terraform validate)
+    (cd $d && rm .terraform.lock.hcl || true && terraform init -upgrade -backend=false && terraform validate)
```
Comment (johanna-skylight) [inline ops/upgradetf.sh:15]: is the `|| true` intentional?

Comment (alismx) [reply ops/upgradetf.sh:15]: It was! I found myself in a situation where I didn't have all the lock files for all the environments, so the command failed. I can revert it now, though. I appreciate the call-out.

**Review by johanna-skylight — APPROVED** [review]
LGTM!

File: ops/upgradetf.sh (hunk 1 above)
Comment (alismx) [reply ops/upgradetf.sh]: So, I passed the `-f` flag here. This should cover my issues and make is simpler to approach any issues with this script.

**Review by fzhao99 — APPROVED** [review]

**Pull request merged**